### PR TITLE
fix(website): automatically update website version from package

### PR DESCRIPTION
**Pull request by TheSisb**
This makes it so that the version number visible on the doc site is pulled directly from the package.json file.  Couldn't figure out [how to || if there was any benefit to] using the GraphQL system to pull this info. This seemed like a reasonable approach so I went with it.  Open to corrections :D

### CODE REVIEW

File: packages/paste-website/gatsby-config.js [hunk 1]
```diff
@@ -58,14 +56,8 @@ module.exports = {
...
         name: 'websiteCore',
-        path: `${__dirname}/src/../`,
-        ignore: [
-          '**/.cache/**',
-          '**/public/**',
-          '**/src/**/*',
-          '**/static/**',
-          '**/types/**',
-        ],
+        path: `${__dirname}/`,
```
Comment (TheSisb) [inline packages/paste-website/gatsby-config.js]: The whole file is prettier except this line, which I just simplified.

File: tsconfig.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "noUnusedParameters": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
+    "resolveJsonModule": true,
```
Comment (TheSisb) [inline tsconfig.json]: Cool feature I turned on in Typescript [that automatically types values from imported JSON files](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html).

Comment (richbachman) [reply tsconfig.json]: 🔥

**Comment by richbachman** [comment]
Implementing GraphQL for the version number seems a bit overkill. I dig the method you've used.

**Comment by SiTaggart** [comment]
Yeah, it's one of those "wrestle data into graphyQL and wrestle data out of graphyQL" or just link to the damn file. 👍

**Pull request merged**